mautic/mautic
# Fixed Import validation when uploading a CSV file in Mautic 4.x with PHP 8

Fixed Import validation when uploading a CSV file in Mautic 4.x with PHP 8

 ## Which branch should I use for my PR? a.b


| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ x]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Make sure you use PHP 8. 
3. Go to Contacts.
4. Select Import from Arrow menu.
5. Upload a CSV file.
7. The upload should be validated successfully.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11011"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
